doc/dev_guide/building_from_source.rst

```diff
@@ -1,283 +1,264 @@
 .. _building_from_source:
 
--------------------------------------------------------------------------------
-                             Building from source
--------------------------------------------------------------------------------
+Building from source
+====================
 
-For downloading Tarantool source and building it, the platforms can differ and
-the preferences can differ. But strategically the steps are always the same.
+To build Tarantool from source files, you will need the following tools:
 
-1. Get tools and libraries that will be necessary for building
-   and testing.
+*   git
+*   ``gcc`` and ``g++``. On Mac OS, this is ``Clang``.
```

Comment:
Nit: `gcc` and `g++` are written as commands, so `Clang` should be `clang` and `clang++` too.